dsync/session.go

```diff
@@ -143,3 +165,18 @@ func randStringBytesMask(n int) string {
 
 	return string(b)
 }
+
+// type blockApiHaser struct {
```

Comment:
remove dead code or if it's mean for debuging maybe separate out into some util file